EarthValley80/EarthValley80/Screen/SideTabbar/SideTabbarLottieView.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  SideTabbarLottieView.swift
+//  EarthValley80
+//
+//  Created by seojeon22 on 2022/11/11.
+//
+
+import Lottie
+import UIKit
+import SwiftUI
+
+struct SideTabbarLottieView: UIViewRepresentable {
+    
+    typealias UIViewType = UIView
+    
+    var filename: String
+    
+    func makeUIView(context: UIViewRepresentableContext<SideTabbarLottieView>) -> UIView {
+        
+        let view = UIView(frame: .zero)
+        let animationView = LottieAnimationView()
+        animationView.animation = Animation.named(filename)
+        animationView.contentMode = .scaleAspectFit
+        animationView.loopMode = .loop
+        animationView.play()
+        
+        animationView.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(animationView)
+        
+        NSLayoutConstraint.activate([
+            animationView.widthAnchor.constraint(equalTo: view.widthAnchor),
+            animationView.heightAnchor.constraint(equalTo: view.heightAnchor)
+        ])
```

Comment:
p3;

이부분을 함수로 따로 분리하는 듀나의 말에 공감함과 동시에, 듀나가 만들어놓은 extension을 사용하는게 어떨까요??
```suggestion
        animationView.constraint(.widthAnchor, constant: view.frame.width)
        animationView.constraint(.heightAnchor, constant: view.frame.height)
```

++).. 근데 .. 적고보니까,,, . 두개 같은... 코드 맞죠..? anchor에 대한 공부를 해봐야겟네요,, 전 이런 상황에서 저렇게 사용하긴했었는데,,,,

Reply:
같은 코드 맞습니당. activate하는 코드가 제가 만든 extension 내부에 있어서 제가 만든 constraint를 사용할 때는 굳이 activate에 관련된 부분 + `translatesAutoresizingMaskIntoConstraints = false` 처리를 안해줘도 됐습니다!

<img width="709" alt="스크린샷 2022-11-14 오전 9 00 21" src="https://user-images.githubusercontent.com/55099365/201551536-8e3d87ad-3d4a-498b-bcb7-f132de0e931c.png">

Reply:
func 으로 처리했습니다 감사합니다~!

bbfdf795e8363cfce9522b031936355a0a089cff